fintech-examples/fintech-server/src/test/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+server.port=8086
```

Comment:
Please switch to yaml property format (It is more widely used in this organization)

Reply:
done